briefcase-forms.rst

```diff
@@ -68,9 +68,9 @@ Pulling from Android 2.x and earlier device
 - Connect your Android device to your computer using a USB cable and choose to mount the drive.
 - Open *ODK Briefcase*.
 - In the window which opens up select :guilabel:`pull` tab.
-- Select `Mounted Android SD Card` in the :guilabel:`pull data from` drop-down.
+- Select `Mounted Android SD Card` in the :guilabel:`pull data from` the drop-down.
```

Comment:
I don't understand this addition.

Reply:
Yes, I misinterpreted the text. I'll correct it!